08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+\textbf{Tied situation:} In a tied situation, there are three possible rules that can be used to eliminate one or more candidates who get the same number of votes. The Condorcet method is one of the voting methods which selects the candidate that dominates other candidates in term of majority relation which ranks the candidates according to one-to-one comparisons. In other words, the Condorcet winner is the candidate that wins a majority in every one-to-one comparison against other candidates. However, Condorcet method is not guaranteed to work when preference lists form a cycle, which is known as the Condorcet paradox or voting paradox. Alternatively, Hare Rule can be used in a tiebreak where all candidates with the least first-place votes will be all eliminated. Moreover, Coombs Rule could be used in place of Hare Rule if the iterative process of removing candidates with the least first-place votes is not preferred. Coombs Rule indicates that the candidate with the most last-placed votes will be removed.
+
+In this coursework, the pairwise Condorcet winner is chosen for a tiebreak setup which states that a candidate who dominates the other candidate in term of majority relation wins.
+
+% Whereas the Hare rule removes all candidates with the least first-place votes and Coombs Rule iteratively removes a candidate with most last placed votes.
+
+% Interestingly, Instant Runoff method is criticized by Condorcet for its disadvantage which involves removing a candidate who is favored by a majority of voters.
+
+\section{Voting Protocol}
+\label{sec:VotingProtocol}
+Voting Protocol is adopted from an example of formal specification given in Robert's Rules of Order (RONR) that defines a series of sequential actions when a voting event happens. This protocol is applicable to any voting method at certain state of the game (i.e. during IIGO session) whenever a voting event happens. With the basis from RONR\footnote{refer to Prof. J Pitt's Draft Book: Self-Organising Multi-Agent Systems (2020) Chapter 4 "Social Choice Theory", Section 4.6.1 Page 114}, the sequence of actions of voting protocol in this game when a voting event happens is as follows (taken an example of voting event for change of rules): the President selects a rule from the rule proposal list to be voted on; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after receiving votes from all islands; the Speaker counts the votes by calling an applicable voting method function for winner selection to produce a result; and finally the Speaker publicly declares the voting result whether the selected rule can be carried out or not. Figure~\ref{fig:RONRVotingProtocol} shows the general sequence of the voting protocol for rules.
```

Comment:
suggested re-writing:

**The suggested voting protocol** is adapted from an example of formal specification .....

This protocol is applicable to any voting method at **a certain** state of the game.

**The sequence of actions of the voting protocol in this game when a voting event happens is as follows: The President selects a rule from the rule proposal list to be voted on**; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after receiving votes from all islands; the Speaker counts the votes by calling an applicable voting method function for winner selection to produce a result; **finally**, the Speaker publicly declares the voting result whether the selected rule can be carried out or not. Figure~\ref{fig:RONRVotingProtocol} shows **the sequence** of the voting protocol for changing and/or adapting a new rule.




can you ask Pitt to how to _formally_ cite his 'book'? @walkeronwater

Reply:
will do